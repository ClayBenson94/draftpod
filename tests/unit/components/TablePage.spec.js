
import { mount } from '@vue/test-utils'
import DraftTable from '../../../src/components/draft/table/DraftTable.vue'

import { testStore } from '../util/test-store'

import { RouterLinkStub } from '@vue/test-utils'

import flushPromises from 'flush-promises'

describe('DraftTable.vue', () => {

  test('draft table displays', () => {

    // mount the page
    const wrapper = mount(DraftTable, {
      sync: false,
      propsData: { draft_id: "2me2Q4mvbpQEJVeMKkuk3y" },
      mocks: {
        $store: testStore()
      },
      stubs: {
        RouterLink: RouterLinkStub
      }
    });

    return flushPromises().then(() => {

      // do a nominal check for some content (this test is really intended to
      // see whether the page renders at all)
      const deckPiles = wrapper.findAll('.pick-list .pile');
      expect(deckPiles).toHaveLength(9);
    });

    
  });

});
