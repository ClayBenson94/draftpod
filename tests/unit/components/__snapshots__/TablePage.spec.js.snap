// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DraftTable.vue draft table displays 1`] = `
<div>
  <nav
    class="navbar navbar-dark navbar-expand-sm user-select-none"
  >
    <img
      class="navbar-logo"
      src="/images/logo.svg"
    />
     
    <span
      class="navbar-brand"
    >
      <a
        class="text-light"
      >
        
      mtglimited.net
    
      </a>
    </span>
     
    <span
      class="navbar-text"
    >
      
      Ravnica Allegiance 
      
      <span>
        
        â€”
        Pack 1, Pick 4
        
        <!---->
      </span>
    </span>
     
    <ul
      class="navbar-nav"
    >
      <li
        class="nav-item"
      >
        <div
          class="dropdown"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="nav-link icon-link dropdown-toggle"
            data-toggle="dropdown"
            href="#"
            id="playersMenuLink"
            title="Players"
          >
            <span
              aria-label="Account Multiple icon"
              class="material-design-icon account-multiple-icon"
              role="img"
            >
              <svg
                class="material-design-icon__svg"
                fill="currentColor"
                height="24"
                viewBox="0 0 24 24"
                width="24"
              >
                <path
                  d="M16,13C15.71,13 15.38,13 15.03,13.05C16.19,13.89 17,15 17,16.5V19H23V16.5C23,14.17 18.33,13 16,13M8,13C5.67,13 1,14.17 1,16.5V19H15V16.5C15,14.17 10.33,13 8,13M8,11C9.66,11 11,9.66 11,8C11,6.34 9.66,5 8,5C6.34,5 5,6.34 5,8C5,9.66 6.34,11 8,11M16,11C17.66,11 19,9.66 19,8C19,6.34 17.66,5 16,5C14.34,5 13,6.34 13,8C13,9.66 14.34,11 16,11Z"
                >
                  <title>
                    Account Multiple icon
                  </title>
                </path>
              </svg>
            </span>
             Players
          
          </a>
           
          <div
            aria-labelledby="playersMenuLink"
            class="dropdown-menu players-menu"
          >
            <div
              class="players"
            >
              <div
                class="players-column players-column-left"
              >
                <div
                  class="player host-player"
                >
                  <div>
                    <div
                      class="player-remove"
                    >
                      <span
                        aria-label="Remove Player from Draft"
                        class="material-design-icon account-remove-icon"
                        role="img"
                      >
                        <svg
                          class="material-design-icon__svg"
                          fill="currentColor"
                          height="24"
                          viewBox="0 0 24 24"
                          width="24"
                        >
                          <path
                            d="M15,14C17.67,14 23,15.33 23,18V20H7V18C7,15.33 12.33,14 15,14M15,12C12.79,12 11,10.21 11,8C11,5.79 12.79,4 15,4C17.21,4 19,5.79 19,8C19,10.21 17.21,12 15,12M5,9.59L7.12,7.46L8.54,8.88L6.41,11L8.54,13.12L7.12,14.54L5,12.41L2.88,14.54L1.46,13.12L3.59,11L1.46,8.88L2.88,7.46L5,9.59Z"
                          >
                            <title>
                              Remove Player from Draft
                            </title>
                          </path>
                        </svg>
                      </span>
                    </div>
                     
                    <div>
                      <span
                        aria-label="Player"
                        class="material-design-icon account-icon"
                        role="img"
                      >
                        <svg
                          class="material-design-icon__svg"
                          fill="currentColor"
                          height="24"
                          viewBox="0 0 24 24"
                          width="24"
                        >
                          <path
                            d="M12,4C14.21,4 16,5.79 16,8C16,10.21 14.21,12 12,12C9.79,12 8,10.21 8,8C8,5.79 9.79,4 12,4M12,14C16.42,14 20,15.79 20,18V20H4V18C4,15.79 7.58,14 12,14Z"
                          >
                            <title>
                              Player
                            </title>
                          </path>
                        </svg>
                      </span>
                    </div>
                     
                    <div
                      class="player-name"
                    >
                      
      Me 
      
                      <div
                        class="pick-number"
                      >
                        
        Pick 4
      
                      </div>
                    </div>
                  </div>
                </div>
                 
                <div
                  class="player host-player"
                >
                  <div>
                    <!---->
                     
                    <div>
                      <span
                        aria-label="Bot"
                        class="material-design-icon account-outline-icon"
                        role="img"
                      >
                        <svg
                          class="material-design-icon__svg"
                          fill="currentColor"
                          height="24"
                          viewBox="0 0 24 24"
                          width="24"
                        >
                          <path
                            d="M12,4C14.21,4 16,5.79 16,8C16,10.21 14.21,12 12,12C9.79,12 8,10.21 8,8C8,5.79 9.79,4 12,4M12,6C10.9,6 10,6.9 10,8C10,9.1 10.9,10 12,10C13.1,10 14,9.1 14,8C14,6.9 13.1,6 12,6M12,13C14.67,13 20,14.33 20,17V20H4V17C4,14.33 9.33,13 12,13M12,14.9C9.03,14.9 5.9,16.36 5.9,17V18.1H18.1V17C18.1,16.36 14.97,14.9 12,14.9Z"
                          >
                            <title>
                              Bot
                            </title>
                          </path>
                        </svg>
                      </span>
                    </div>
                     
                    <div
                      class="player-colors"
                    >
                      <div
                        class="player-name"
                      >
                        
        Bot
      
                      </div>
                    </div>
                  </div>
                </div>
                 
                <div
                  class="player host-player"
                >
                  <div>
                    <!---->
                     
                    <div>
                      <span
                        aria-label="Bot"
                        class="material-design-icon account-outline-icon"
                        role="img"
                      >
                        <svg
                          class="material-design-icon__svg"
                          fill="currentColor"
                          height="24"
                          viewBox="0 0 24 24"
                          width="24"
                        >
                          <path
                            d="M12,4C14.21,4 16,5.79 16,8C16,10.21 14.21,12 12,12C9.79,12 8,10.21 8,8C8,5.79 9.79,4 12,4M12,6C10.9,6 10,6.9 10,8C10,9.1 10.9,10 12,10C13.1,10 14,9.1 14,8C14,6.9 13.1,6 12,6M12,13C14.67,13 20,14.33 20,17V20H4V17C4,14.33 9.33,13 12,13M12,14.9C9.03,14.9 5.9,16.36 5.9,17V18.1H18.1V17C18.1,16.36 14.97,14.9 12,14.9Z"
                          >
                            <title>
                              Bot
                            </title>
                          </path>
                        </svg>
                      </span>
                    </div>
                     
                    <div
                      class="player-colors"
                    >
                      <div
                        class="player-name"
                      >
                        
        Bot
      
                      </div>
                    </div>
                  </div>
                </div>
                 
                <div
                  class="player host-player"
                >
                  <div>
                    <!---->
                     
                    <div>
                      <span
                        aria-label="Bot"
                        class="material-design-icon account-outline-icon"
                        role="img"
                      >
                        <svg
                          class="material-design-icon__svg"
                          fill="currentColor"
                          height="24"
                          viewBox="0 0 24 24"
                          width="24"
                        >
                          <path
                            d="M12,4C14.21,4 16,5.79 16,8C16,10.21 14.21,12 12,12C9.79,12 8,10.21 8,8C8,5.79 9.79,4 12,4M12,6C10.9,6 10,6.9 10,8C10,9.1 10.9,10 12,10C13.1,10 14,9.1 14,8C14,6.9 13.1,6 12,6M12,13C14.67,13 20,14.33 20,17V20H4V17C4,14.33 9.33,13 12,13M12,14.9C9.03,14.9 5.9,16.36 5.9,17V18.1H18.1V17C18.1,16.36 14.97,14.9 12,14.9Z"
                          >
                            <title>
                              Bot
                            </title>
                          </path>
                        </svg>
                      </span>
                    </div>
                     
                    <div
                      class="player-colors"
                    >
                      <div
                        class="player-name"
                      >
                        
        Bot
      
                      </div>
                    </div>
                  </div>
                </div>
              </div>
               
              <div
                class="players-table"
              >
                <div
                  class="pack-number"
                >
                  <svg
                    class="player-arrow"
                    height="24"
                    style="top: 3px; right: 1px;"
                    viewBox="0 0 224 224"
                    width="24"
                    x="0px"
                    xmlns="http://www.w3.org/2000/svg"
                    y="0px"
                  >
                    <g
                      fill-rule="nonzero"
                      font-family="none"
                      font-size="none"
                      font-weight="none"
                      stroke="none"
                      stroke-dasharray=""
                      stroke-dashoffset="0"
                      stroke-linecap="butt"
                      stroke-linejoin="miter"
                      stroke-miterlimit="10"
                      stroke-width="1"
                      text-anchor="none"
                    >
                      <path
                        d="M0,224v-224h224v224z"
                        fill="none"
                      />
                       
                      <g
                        fill="#aaaaaa"
                      >
                        <path
                          d="M28,28v18.66667h28c46.5042,0 84,37.4958 84,84v18.66667h-37.33333l46.66667,46.66667l46.66667,-46.66667h-37.33333v-18.66667c0,-56.5918 -46.07487,-102.66667 -102.66667,-102.66667z"
                        />
                         
                        <!---->
                         
                        <!---->
                         
                        <!---->
                      </g>
                    </g>
                  </svg>
                   
                  <svg
                    class="player-arrow"
                    height="24"
                    style="bottom: 4px; left: 2px;"
                    viewBox="0 0 224 224"
                    width="24"
                    x="0px"
                    xmlns="http://www.w3.org/2000/svg"
                    y="0px"
                  >
                    <g
                      fill-rule="nonzero"
                      font-family="none"
                      font-size="none"
                      font-weight="none"
                      stroke="none"
                      stroke-dasharray=""
                      stroke-dashoffset="0"
                      stroke-linecap="butt"
                      stroke-linejoin="miter"
                      stroke-miterlimit="10"
                      stroke-width="1"
                      text-anchor="none"
                    >
                      <path
                        d="M0,224v-224h224v224z"
                        fill="none"
                      />
                       
                      <g
                        fill="#aaaaaa"
                      >
                        <!---->
                         
                        <path
                          d="M74.66667,28l-46.66667,46.66667h37.33333v18.66667c0,56.5918 46.07487,102.66667 102.66667,102.66667h28v-18.66667h-28c-46.5042,0 -84,-37.4958 -84,-84v-18.66667h37.33333z"
                        />
                         
                        <!---->
                         
                        <!---->
                      </g>
                    </g>
                  </svg>
                   
                  <!---->
                   
                  <!---->
                  
      Pack 1
    
                </div>
                 
                <a
                  class="show-bot-colors"
                >
                  
      Show bot colors
    
                </a>
              </div>
               
              <div
                class="players-column players-column-right"
              >
                <div
                  class="player host-player"
                >
                  <div>
                    <!---->
                     
                    <div>
                      <span
                        aria-label="Bot"
                        class="material-design-icon account-outline-icon"
                        role="img"
                      >
                        <svg
                          class="material-design-icon__svg"
                          fill="currentColor"
                          height="24"
                          viewBox="0 0 24 24"
                          width="24"
                        >
                          <path
                            d="M12,4C14.21,4 16,5.79 16,8C16,10.21 14.21,12 12,12C9.79,12 8,10.21 8,8C8,5.79 9.79,4 12,4M12,6C10.9,6 10,6.9 10,8C10,9.1 10.9,10 12,10C13.1,10 14,9.1 14,8C14,6.9 13.1,6 12,6M12,13C14.67,13 20,14.33 20,17V20H4V17C4,14.33 9.33,13 12,13M12,14.9C9.03,14.9 5.9,16.36 5.9,17V18.1H18.1V17C18.1,16.36 14.97,14.9 12,14.9Z"
                          >
                            <title>
                              Bot
                            </title>
                          </path>
                        </svg>
                      </span>
                    </div>
                     
                    <div
                      class="player-colors"
                    >
                      <div
                        class="player-name"
                      >
                        
        Bot
      
                      </div>
                    </div>
                  </div>
                </div>
                 
                <div
                  class="player host-player"
                >
                  <div>
                    <!---->
                     
                    <div>
                      <span
                        aria-label="Bot"
                        class="material-design-icon account-outline-icon"
                        role="img"
                      >
                        <svg
                          class="material-design-icon__svg"
                          fill="currentColor"
                          height="24"
                          viewBox="0 0 24 24"
                          width="24"
                        >
                          <path
                            d="M12,4C14.21,4 16,5.79 16,8C16,10.21 14.21,12 12,12C9.79,12 8,10.21 8,8C8,5.79 9.79,4 12,4M12,6C10.9,6 10,6.9 10,8C10,9.1 10.9,10 12,10C13.1,10 14,9.1 14,8C14,6.9 13.1,6 12,6M12,13C14.67,13 20,14.33 20,17V20H4V17C4,14.33 9.33,13 12,13M12,14.9C9.03,14.9 5.9,16.36 5.9,17V18.1H18.1V17C18.1,16.36 14.97,14.9 12,14.9Z"
                          >
                            <title>
                              Bot
                            </title>
                          </path>
                        </svg>
                      </span>
                    </div>
                     
                    <div
                      class="player-colors"
                    >
                      <div
                        class="player-name"
                      >
                        
        Bot
      
                      </div>
                    </div>
                  </div>
                </div>
                 
                <div
                  class="player host-player"
                >
                  <div>
                    <!---->
                     
                    <div>
                      <span
                        aria-label="Bot"
                        class="material-design-icon account-outline-icon"
                        role="img"
                      >
                        <svg
                          class="material-design-icon__svg"
                          fill="currentColor"
                          height="24"
                          viewBox="0 0 24 24"
                          width="24"
                        >
                          <path
                            d="M12,4C14.21,4 16,5.79 16,8C16,10.21 14.21,12 12,12C9.79,12 8,10.21 8,8C8,5.79 9.79,4 12,4M12,6C10.9,6 10,6.9 10,8C10,9.1 10.9,10 12,10C13.1,10 14,9.1 14,8C14,6.9 13.1,6 12,6M12,13C14.67,13 20,14.33 20,17V20H4V17C4,14.33 9.33,13 12,13M12,14.9C9.03,14.9 5.9,16.36 5.9,17V18.1H18.1V17C18.1,16.36 14.97,14.9 12,14.9Z"
                          >
                            <title>
                              Bot
                            </title>
                          </path>
                        </svg>
                      </span>
                    </div>
                     
                    <div
                      class="player-colors"
                    >
                      <div
                        class="player-name"
                      >
                        
        Bot
      
                      </div>
                    </div>
                  </div>
                </div>
                 
                <div
                  class="player host-player"
                >
                  <div>
                    <!---->
                     
                    <div>
                      <span
                        aria-label="Bot"
                        class="material-design-icon account-outline-icon"
                        role="img"
                      >
                        <svg
                          class="material-design-icon__svg"
                          fill="currentColor"
                          height="24"
                          viewBox="0 0 24 24"
                          width="24"
                        >
                          <path
                            d="M12,4C14.21,4 16,5.79 16,8C16,10.21 14.21,12 12,12C9.79,12 8,10.21 8,8C8,5.79 9.79,4 12,4M12,6C10.9,6 10,6.9 10,8C10,9.1 10.9,10 12,10C13.1,10 14,9.1 14,8C14,6.9 13.1,6 12,6M12,13C14.67,13 20,14.33 20,17V20H4V17C4,14.33 9.33,13 12,13M12,14.9C9.03,14.9 5.9,16.36 5.9,17V18.1H18.1V17C18.1,16.36 14.97,14.9 12,14.9Z"
                          >
                            <title>
                              Bot
                            </title>
                          </path>
                        </svg>
                      </span>
                    </div>
                     
                    <div
                      class="player-colors"
                    >
                      <div
                        class="player-name"
                      >
                        
        Bot
      
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </li>
       
      <!---->
    </ul>
     
    <ul
      class="navbar-nav ml-auto"
    >
      <li
        class="nav-item"
      >
        <a
          class="nav-link"
        >
          
        Home
      
        </a>
      </li>
       
      <li
        class="nav-item"
      >
        <a
          class="nav-link"
        >
          
        Play
      
        </a>
      </li>
       
      <li
        class="nav-item"
      >
        <a
          class="nav-link"
        >
          
        Guide
      
        </a>
      </li>
       
      <li
        class="nav-item"
      >
        <a
          class="nav-link icon-link"
          href="https://twitter.com/DraftpodMTG"
        >
          <span
            aria-label="Twitter icon"
            class="material-design-icon twitter-icon"
            role="img"
          >
            <svg
              class="material-design-icon__svg"
              fill="currentColor"
              height="24"
              viewBox="0 0 24 24"
              width="24"
            >
              <path
                d="M22.46,6C21.69,6.35 20.86,6.58 20,6.69C20.88,6.16 21.56,5.32 21.88,4.31C21.05,4.81 20.13,5.16 19.16,5.36C18.37,4.5 17.26,4 16,4C13.65,4 11.73,5.92 11.73,8.29C11.73,8.63 11.77,8.96 11.84,9.27C8.28,9.09 5.11,7.38 3,4.79C2.63,5.42 2.42,6.16 2.42,6.94C2.42,8.43 3.17,9.75 4.33,10.5C3.62,10.5 2.96,10.3 2.38,10C2.38,10 2.38,10 2.38,10.03C2.38,12.11 3.86,13.85 5.82,14.24C5.46,14.34 5.08,14.39 4.69,14.39C4.42,14.39 4.15,14.36 3.89,14.31C4.43,16 6,17.26 7.89,17.29C6.43,18.45 4.58,19.13 2.56,19.13C2.22,19.13 1.88,19.11 1.54,19.07C3.44,20.29 5.7,21 8.12,21C16,21 20.33,14.46 20.33,8.79C20.33,8.6 20.33,8.42 20.32,8.23C21.16,7.63 21.88,6.87 22.46,6Z"
              >
                <title>
                  Twitter icon
                </title>
              </path>
            </svg>
          </span>
        </a>
      </li>
    </ul>
  </nav>
   
  <div
    class="draft-page"
  >
    <!---->
     
    <div
      class="draft-cards user-select-none"
    >
      <div
        class="card pack-panel card-select-panel"
        name="pack-hide"
      >
        <!---->
         
        <div
          class="card-body"
        >
          <div
            class="card-select-container"
          >
            <span
              class="mtgcard mtgcard-draggable"
              draggable="true"
            >
              <img
                class=""
                src="https://img.scryfall.com/cards/normal/front/f/9/f97a6d34-03ab-49f1-b02e-405b733f8843.jpg?1549415203"
              />
               
              <!---->
               
              <div
                style="position: fixed; top: -1000px;"
              />
            </span>
            <span
              class="mtgcard mtgcard-draggable"
              draggable="true"
            >
              <img
                class=""
                src="https://img.scryfall.com/cards/normal/front/0/0/00320106-ce51-46a9-b0f9-79b3baf4e505.jpg?1549415058"
              />
               
              <!---->
               
              <div
                style="position: fixed; top: -1000px;"
              />
            </span>
            <span
              class="mtgcard mtgcard-draggable"
              draggable="true"
            >
              <img
                class=""
                src="https://img.scryfall.com/cards/normal/front/7/8/78cc4e02-949f-4879-9f32-7c33490d0b45.jpg?1549414941"
              />
               
              <!---->
               
              <div
                style="position: fixed; top: -1000px;"
              />
            </span>
            <span
              class="mtgcard mtgcard-draggable"
              draggable="true"
            >
              <img
                class=""
                src="https://img.scryfall.com/cards/normal/front/1/e/1ec78880-a8ec-4c87-bc3f-e2a79d154884.jpg?1549415114"
              />
               
              <!---->
               
              <div
                style="position: fixed; top: -1000px;"
              />
            </span>
            <span
              class="mtgcard mtgcard-draggable"
              draggable="true"
            >
              <img
                class=""
                src="https://img.scryfall.com/cards/normal/front/7/c/7cd03b99-d62e-428a-9e0d-097f1227a4da.jpg?1549414251"
              />
               
              <!---->
               
              <div
                style="position: fixed; top: -1000px;"
              />
            </span>
            <span
              class="mtgcard mtgcard-draggable"
              draggable="true"
            >
              <img
                class=""
                src="https://img.scryfall.com/cards/normal/front/b/0/b075a6f2-5196-45e9-b062-f131f7b1a347.jpg?1549413860"
              />
               
              <!---->
               
              <div
                style="position: fixed; top: -1000px;"
              />
            </span>
            <span
              class="mtgcard mtgcard-draggable"
              draggable="true"
            >
              <img
                class=""
                src="https://img.scryfall.com/cards/normal/front/8/b/8bbab274-69dd-44a9-9310-a15779c35cad.jpg?1549414638"
              />
               
              <!---->
               
              <div
                style="position: fixed; top: -1000px;"
              />
            </span>
            <span
              class="mtgcard mtgcard-draggable"
              draggable="true"
            >
              <img
                class=""
                src="https://img.scryfall.com/cards/normal/front/1/3/13aed078-9e29-48e7-b145-5252362031a0.jpg?1549415096"
              />
               
              <!---->
               
              <div
                style="position: fixed; top: -1000px;"
              />
            </span>
            <span
              class="mtgcard mtgcard-draggable"
              draggable="true"
            >
              <img
                class=""
                src="https://img.scryfall.com/cards/normal/front/8/a/8ace095a-3ea9-4121-8ffb-5b3612b96985.jpg?1549414456"
              />
               
              <!---->
               
              <div
                style="position: fixed; top: -1000px;"
              />
            </span>
            <span
              class="mtgcard mtgcard-draggable"
              draggable="true"
            >
              <img
                class=""
                src="https://img.scryfall.com/cards/normal/front/6/3/63d566fc-0936-4035-96fd-f8b0c4eadbf5.jpg?1549413865"
              />
               
              <!---->
               
              <div
                style="position: fixed; top: -1000px;"
              />
            </span>
            <span
              class="mtgcard mtgcard-draggable"
              draggable="true"
            >
              <img
                class=""
                src="https://img.scryfall.com/cards/normal/front/b/6/b6263410-20c5-43b4-8183-3c02c10d07fb.jpg?1549414145"
              />
               
              <!---->
               
              <div
                style="position: fixed; top: -1000px;"
              />
            </span>
            <span
              class="mtgcard mtgcard-draggable"
              draggable="true"
            >
              <img
                class=""
                src="https://img.scryfall.com/cards/normal/front/c/b/cba5fb67-e161-4e89-be3e-c8021122ff19.jpg?1549415208"
              />
               
              <!---->
               
              <div
                style="position: fixed; top: -1000px;"
              />
            </span>
          </div>
        </div>
      </div>
       
      <div
        class="pick-panel card bg-white"
      >
        <div
          class="card-header tabs-header"
        >
          <ul
            class="nav"
            role="tablist"
          >
            <li
              class="nav-item"
            >
              <a
                aria-controls="deck"
                aria-selected="true"
                class="nav-link active"
                data-toggle="tab"
                href="#deck"
                id="deck-tab"
                role="tab"
              >
                
          Picks
        
              </a>
            </li>
             
            <!---->
          </ul>
        </div>
         
        <div
          class="card-body tab-content"
        >
          <div
            aria-labelledby="deck-tab"
            class="tab-pane fade show active"
            id="deck"
            role="tabpanel"
          >
            <div
              class="pick-list"
            >
              <div
                class="pile"
              >
                <!---->
                  
                <div
                  class="pile-controls"
                  style="margin-top: -10%;"
                />
                 
                <div
                  class="pile-drag-insert"
                  style="margin-top: 0px; display: none;"
                />
              </div>
              <div
                class="pile"
              >
                <!---->
                  
                <div
                  class="pile-controls"
                  style="margin-top: -10%;"
                />
                 
                <div
                  class="pile-drag-insert"
                  style="margin-top: 0px; display: none;"
                />
              </div>
              <div
                class="pile"
              >
                <!---->
                  
                <div
                  class="pile-controls"
                  style="margin-top: -10%;"
                />
                 
                <div
                  class="pile-drag-insert"
                  style="margin-top: 0px; display: none;"
                />
              </div>
              <div
                class="pile"
              >
                <!---->
                 
                <span
                  class="mtgcard mtgcard-draggable"
                  draggable="true"
                  style="margin-top: 0%;"
                >
                  <img
                    class=""
                    src="https://img.scryfall.com/cards/normal/front/5/8/58f70b8a-62ec-4ae6-97d2-3c6ae86a3602.jpg?1549414498"
                  />
                   
                  <!---->
                   
                  <div
                    style="position: fixed; top: -1000px;"
                  />
                </span>
                 
                <div
                  class="pile-controls"
                  style="margin-top: 140%;"
                />
                 
                <div
                  class="pile-drag-insert"
                  style="margin-top: 0px; display: none;"
                />
              </div>
              <div
                class="pile"
              >
                <!---->
                  
                <div
                  class="pile-controls"
                  style="margin-top: -10%;"
                />
                 
                <div
                  class="pile-drag-insert"
                  style="margin-top: 0px; display: none;"
                />
              </div>
              <div
                class="pile"
              >
                <!---->
                  
                <div
                  class="pile-controls"
                  style="margin-top: -10%;"
                />
                 
                <div
                  class="pile-drag-insert"
                  style="margin-top: 0px; display: none;"
                />
              </div>
              <div
                class="pile"
              >
                <!---->
                 
                <span
                  class="mtgcard mtgcard-draggable"
                  draggable="true"
                  style="margin-top: 0%;"
                >
                  <img
                    class=""
                    src="https://img.scryfall.com/cards/normal/front/8/4/84062ce2-fea2-4e06-b83b-7cc597fb2a1b.jpg?1549414936"
                  />
                   
                  <!---->
                   
                  <div
                    style="position: fixed; top: -1000px;"
                  />
                </span>
                <span
                  class="mtgcard mtgcard-draggable"
                  draggable="true"
                  style="margin-top: 16%;"
                >
                  <img
                    class=""
                    src="https://img.scryfall.com/cards/normal/front/9/7/97295660-6bea-46ae-9a3b-0fc6abba407f.jpg?1549414483"
                  />
                   
                  <!---->
                   
                  <div
                    style="position: fixed; top: -1000px;"
                  />
                </span>
                 
                <div
                  class="pile-controls"
                  style="margin-top: 156%;"
                />
                 
                <div
                  class="pile-drag-insert"
                  style="margin-top: 0px; display: none;"
                />
              </div>
               
              <div
                class="pile pile-separator"
              />
               
              <div
                class="pile"
              >
                <div
                  class="pile-caption"
                  style="text-align: center;"
                >
                  
    Sideboard
                  <!---->
                </div>
                  
                <div
                  class="pile-controls"
                  style="margin-top: 6%;"
                />
                 
                <div
                  class="pile-drag-insert"
                  style="margin-top: 0px; display: none;"
                />
              </div>
            </div>
          </div>
           
          <!---->
        </div>
      </div>
    </div>
     
    <div
      class="infobar user-select-none"
    >
      <div
        class="card-viewer"
      >
        <img
          src="/images/card-back.png"
        />
      </div>
       
      <!---->
       
      <div
        class="card deck-stats"
      >
        <div
          class="card-header"
        >
          <div
            class="float-left header-text text-light"
          >
            
      Cards
    
          </div>
           
          <div
            class="float-left"
          />
           
          <div
            class="float-right"
          />
        </div>
         
        <div
          class="card-body"
        >
          <div
            class="mana-curve-legend"
          >
            <span
              class="mana-key creatures-key bg-danger"
            >
              
Â 
    
            </span>
             Creature (1)
    
            <span
              class="mana-key other-key bg-info"
            >
              
Â 
    
            </span>
             Other (2)
  
          </div>
           
          <div
            class="mana-curve ct-chart ct-perfect-fourth"
          />
           
          <table
            class="table table-sm deck-colors"
          >
            <tbody>
              <tr>
                <td
                  width="60%"
                >
                  <img
                    src="/images/mana-red.svg"
                    width="18"
                  />
                   Mountain
      
                </td>
                 
                <td
                  align="right"
                >
                  
        3
      
                </td>
                 
                <td
                  align="right"
                >
                  
        75%
      
                </td>
              </tr>
              <tr>
                <td
                  width="60%"
                >
                  <img
                    src="/images/mana-green.svg"
                    width="18"
                  />
                   Forest
      
                </td>
                 
                <td
                  align="right"
                >
                  
        1
      
                </td>
                 
                <td
                  align="right"
                >
                  
        25%
      
                </td>
              </tr>
              <tr>
                <td
                  width="60%"
                >
                  <img
                    src="/images/mana-white.svg"
                    width="18"
                  />
                   Plains
      
                </td>
                 
                <td
                  align="right"
                >
                  
        0
      
                </td>
                 
                <td
                  align="right"
                >
                  
        0%
      
                </td>
              </tr>
              <tr>
                <td
                  width="60%"
                >
                  <img
                    src="/images/mana-black.svg"
                    width="18"
                  />
                   Swamp
      
                </td>
                 
                <td
                  align="right"
                >
                  
        0
      
                </td>
                 
                <td
                  align="right"
                >
                  
        0%
      
                </td>
              </tr>
              <tr>
                <td
                  width="60%"
                >
                  <img
                    src="/images/mana-blue.svg"
                    width="18"
                  />
                   Island
      
                </td>
                 
                <td
                  align="right"
                >
                  
        0
      
                </td>
                 
                <td
                  align="right"
                >
                  
        0%
      
                </td>
              </tr>
              <tr>
                <td
                  width="60%"
                >
                  <img
                    src="/images/mana-colorless.svg"
                    width="18"
                  />
                   Colorless
      
                </td>
                 
                <td
                  align="right"
                >
                  
        0
      
                </td>
                 
                <td
                  align="right"
                >
                  
        0%
      
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </div>
  </div>
</div>
`;
